lib/trento/domain/sap_system/sap_system.ex

```diff
@@ -234,6 +240,9 @@ defmodule Trento.Domain.SapSystem do
     |> Multi.execute(fn _ ->
       maybe_emit_database_instance_health_changed_event(instance, command)
     end)
+    |> Multi.execute(fn _ ->
```

Comment:
I think this function should come after the `maybe_emit_database_instance_registered_event`.
It looks more logical, and we spare an unnecessary event emission (this one, as the next event is putting this value to false in this case in the 1st registration, right)